sunpy/map/map_factory.py

```diff
@@ -79,6 +77,17 @@ class MapFactory(BasicRegistrationFactory):
 
     >>> mymap = sunpy.map.Map(data, header)   # doctest: +SKIP
 
+    * data, wcs object, in tuple
+    >>> from astropy.wcs import WCS
+    >>> wcs = WCS(AIA_171_IMAGE)
+    >>> mymap = sunpy.map.Map((data, wcs))    #doctest: +SKIP
+    
+    * data, wcs object, not in tuple
+    
+    >>> from astropy.wcs import WCS
+    >>> wcs = WCS(AIA_171_IMAGE)
+    >>> mymap = sunpy.map.Map(data, wcs)    #doctest: +SKIP
```

Comment:
If possible, I would rather these tests do not get skipped.

Reply:
It would require a change in test_map_factory.py?

Reply:
Sorry, I don't follow.